.github/workflows/build-riscv64.yml

```diff
@@ -40,7 +40,7 @@ jobs:
             -v ${{ github.workspace }}:/ws --workdir=/ws \
             chianetwork/ubuntu-22.04-risc-builder:latest \
             bash -exc '\
-            cargo test --release
+            cargo test --release -p chia-consensus
```

Comment:
I wonder if it was deliberate to omit the `--all`, and really only run these tests

Reply:
chia-ssl doesn't compile on riscv